database/sql/src/main/scala/cromwell/database/sql/WorkflowStoreSqlDatabase.scala

```diff
@@ -40,15 +40,13 @@ ____    __    ____  ______   .______       __  ___  _______  __        ______
     * @param workflowStateToDelete2 Also delete rows with this state.
     * @param workflowStateForUpdate Update other rows to this state.
     * @param ec                     The execution context where to run asynchronous operations.
-    * @return Number of rows deleted, plus Some(heartbeatTimestampIsEmpty) if the workflow exists in the store, where
-    *         heartbeatTimestampIsEmpty is true if the workflow's heartbeat timestamp is empty, and None if the workflow
-    *         does not exist.
+    * @return None if no rows were matched, Some(true) if a row was deleted or Some(false) if a row was updated.
```

Comment:
It sort of feels to me like a set of three well-named `case object`s would be better than this complicated encoding of Options and Booleans?